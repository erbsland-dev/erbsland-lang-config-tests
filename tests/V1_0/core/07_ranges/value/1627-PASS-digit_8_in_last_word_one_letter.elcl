# Erbsland Configuration Language Test File
[main]
name_8: 12