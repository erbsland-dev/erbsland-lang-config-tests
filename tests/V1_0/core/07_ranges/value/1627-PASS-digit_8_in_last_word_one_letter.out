main = SectionWithNames()
main.name_8 = Integer(12)
